vsce/server/src/server.ts

```diff
@@ -558,15 +563,29 @@ CallStack (from HasCallStack):
     connection.console.log(`Found pattern: ${m}`);
 
     // ERROR MESSAGE m: error: <json>
+    console.info();
+    console.info('Compilation errors string:');
+    console.info(compileErrors);
     const errorJson: ReachCompilerErrorJSON = JSON.parse(m[0].substring(7));
 
     //connection.console.log(`Tokens: ` + tokens);
-    const linePos = errorJson.ce_position[0] - 1;
-    const charPos = errorJson.ce_position[1] - 1;
+    let linePos = errorJson.ce_position[0] - 1;
+    let charPos = errorJson.ce_position[1] - 1;
     const suggestions = errorJson.ce_suggestions;
     const actualMessage = errorJson.ce_errorMessage;
     const offendingToken = errorJson.ce_offendingToken;
     const reachCompilerErrorCode = errorJson.ce_errorCode;
+    if (reachCompilerErrorCode === "RE0089") {
+      const { ce_stackTrace } = errorJson;
+      // Match "...at (/app/index.rsh:8:12:application)".
+      const regexp: RegExp = /index.rsh:(\d+):(\d+):.+$/;
```

Comment:
Is there a way to ask the extension for "current file name" so that this can work for more than just `index.rsh`?

Reply:
So, the `index.rsh` here isn't actually an `index.rsh` on the user's local device: it's a temporary file created in a temporary directory that we compile.

Reply:
![image](https://user-images.githubusercontent.com/43425812/182437615-c7596f5b-b9d1-41ea-a81c-fe1884326542.png)